src/main/java/com/cinema/model/User.java

```diff
@@ -0,0 +1,33 @@
+package com.cinema.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NonNull
+    @Column(unique = true)
+    private String email;
+    @NonNull
+    private String password;
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
```

Comment:
what the purpose of EqualsAndHashCode.Exclude and ToString.Exclude annotations ?

Reply:
I wanted to exclude field "password" from this methods because I suppose password / hash shouldn't be shown and used in equals